mmseg/models/segmentors/encoder_decoder.py

```diff
@@ -300,3 +300,23 @@ def aug_test(self, imgs, img_metas, rescale=True):
         # unravel batch dim
         seg_pred = list(seg_pred)
         return seg_pred
+
+    def aug_test_logits(self, img, img_metas, rescale=True):
+        """
+            Test with augmentations. 
+            Return seg_map logits.
+            Only rescale=True is supported.
+        """
+        # aug_test rescale all imgs back to ori_shape for now
+        assert rescale
+        # to save memory, we get augmented seg logit inplace
+
+        imgs = img
+        seg_logit = self.inference(imgs[0], img_metas[0], rescale)
+        for i in range(1, len(imgs)):
+            cur_seg_logit = self.inference(imgs[i], img_metas[i], rescale)
+            seg_logit += cur_seg_logit
```

Comment:
I have a little question why not divide `seg_logit` by `len(imgs)` that is the number of augmented data

Reply:
yes, the `seg_logit` should divid `len(imgs)`, i have a mistake, even it will have same result with `divid len(imgs)` by `torch.argmax`, but `divid len(imgs)` is better.

By the way, do you think this needs to be added to this repo? Or are you willing to accept this idea (add model_ensemble code to repo)?

Reply:
Would you mind providing your email to us? It is convenient for us to contact offline

Reply:
> Would you mind providing your email to us? It is convenient for us to contact offline



> commented

`22151338@zju.edu.cn`

Reply:
Please check your email, I have sent our invitation.